### PR TITLE
Added a cmd to list course users using the new cmd infrastructure.

### PR DESCRIPTION
**Pull request by OliverLok**

### CODE REVIEW

**Review by Lucas-Ellenberger — CHANGES_REQUESTED** [review]
The new CMD testing infrastructure looks sweet!

Clean PR, only one small note.

**Review by Lucas-Ellenberger — APPROVED** [review]
lgtm!

File: cmd/course-user-list/main.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	cmd.MustHandleCMDRequestAndExitFull(`courses/users/list`, request, users.ListResponse{}, args.CommonOptions, printFunc)
+}
+
+func listCourseUsersTable(response core.APIResponse) string {
+	var responseContent users.ListResponse
+	util.MustJSONFromString(util.MustToJSON(response.Content), &responseContent)
+
+	var courseUsersTable strings.Builder
+
+	headers := []string{"email", "name", "course-role", "LMSID"}
```
Comment (eriq-augustine) [inline cmd/course-user-list/main.go:54]: Make this consistent with the Python interface:
https://github.com/edulinq/autograder-py/blob/main/autograder/cli/common.py#L3

This is not something that is critical, but is nice to have and easy to implement.

**Review by eriq-augustine — CHANGES_REQUESTED** [review]

File: cmd/course-user-list/main.go (hunk 1 above)
Comment (OliverLok) [reply cmd/course-user-list/main.go:54]: Are you referring to making the header names consistent?

Comment (eriq-augustine) [reply cmd/course-user-list/main.go:54]: Yeah.

**Review by eriq-augustine — APPROVED** [review]
lgtm!

**Pull request merged**